src/CTA.Rules.Update/CodeReplacer.cs

```diff
@@ -271,6 +275,9 @@ public List<GenericActionExecution> ValidateActions(List<GenericActionExecution>
                 {
                     var actionValidationException = new ActionValidationException(action.Key, action.Name);
                     action.InvalidExecutions = 1;
+                    var originalAction = actions.FirstOrDefault(a => a.GetHashCode() == action.GetHashCode());
+                    if(originalAction != null) 
+                        originalAction.InvalidExecutions = 1;
```

Comment:
Why do we need this?

Reply:
It was modifying the original actions list here to set the invalid executions if needed. Iterating over a new reduced list that doesn't get returned I need to access original list that gets returned to modify the corresponding element.

Reply:
Sorry this was my mistake, it doe sindeed pass by reference and this code is superfluous and not needed, will remove.